integration_test/sendL2msg.test.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021, Offchain Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-env node */
+// import { instantiateBridge } from './instantiate_bridge'
+;('use strict')
+
+import { BigNumber, ethers, providers, Wallet } from "ethers";
+import { InboxTools } from "../src/lib/inbox/inbox"
+import { getL2Network } from '../src/lib/dataEntities/networks'
+import { testSetup } from '../scripts/testSetup'
+import { expect } from "chai";
+
+const sendSignedTx = async () => {
+    const { l1Deployer,l2Deployer } = await testSetup()
+    const l2Network = await getL2Network(await l2Deployer.getChainId())
+    const inbox = new InboxTools(l1Deployer, l2Network)
+    const message = {
+        to: await l2Deployer.getAddress(),
+        value: BigNumber.from(0),
+        data: "0x12"
+    }
+    const signedTx = await inbox.signL2Message(message, l2Deployer)
+    const l1TxReceipt = await inbox.sendL2SignedMessage(signedTx)
+    return {
+        signedMsg: signedTx,
+        l1TransactionReceipt: l1TxReceipt
+    }
+}
+
+describe("Send signedTx to l2 using inbox", () => {
```

Comment:
We should also add a test for when the l2gasprice is set to low, and another one for contract deployment

Reply:
If l2 gas price is lower than the base fee, network will ignore that tx so it won't record on chain.

Reply:
I think that would be a decent test. You could have a test that does
1. Send a tx with gas price too low
2. Send a tx with gas price high enough
3. Wait until tx2 is mined
4. Check that tx1 was never executed on the L2 chain by looking for it's transaction hash